### DISCUSSION TITLE
Stationary State Confidence Intervals

### DISCUSSION THREAD

**Discussion by physalus** [TheoMichelot/moveHMM#27] [Q&A] [answered]
The plotStationary function has the option to include the 95% CI. Is there a way to extract the CI to be used in a custom plot?

**Comment by TheoMichelot** [comment, marked as answer]
Hi James,

There is no convenient function to do this, unfortunately, because the confidence intervals are directly computed in the `plotStationary` function. Your best option might be to copy [the internal code of plotStationary](https://github.com/TheoMichelot/moveHMM/blob/master/R/plotStationary.R), and use the matrices `lci` and `uci` to create your plots. `lci` is a matrix of lower bounds (with one column for each state), and `uci` is a similar matrix for the upper bounds.

Let me know if anything is unclear.

Théo
